src/sentry/migrations/0645_backfill_add_uuid_to_all_rule_actions.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 5.0.2 on 2024-02-13 20:45
+from django.db import migrations
+
+from sentry.api.serializers.rest_framework import ensure_action_uuid
```

Comment:
Other than things like `RangeQuerySetWrapperWithProgressBarApprox` which are migration specific, we generally try to avoid importing code here. The reason for this is that if the code changes, the migration can break in unexpected ways.

Generally, we recommend just copy/pasting that external code into the migration so that it's stable. If the code is extremely complicated we can make exceptions, so it's not a hard rule. In this case though it should be simple enough.

Reply:
Got it, makes sense! Thank you for letting me know, will remember this for next time